<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPageForm</class>
 <widget class="QWidget" name="MainPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HEY</string>
  </property>
  <widget class="QToolButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>671</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome back, </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>491</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="MainPage" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>590</height>
    </rect>
   </property>
  </widget>
  <zorder>MainPage</zorder>
  <zorder>menuButton</zorder>
  <zorder>welcomeLabel</zorder>
  <zorder>gridLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
